app/routes/prospective_intern_routes.py

```
@@ -1,5 +1,5 @@
 from flask import Blueprint, render_template, request, redirect, url_for, jsonify, flash
-from flask_jwt_extended import get_jwt_identity
+from flask_jwt_extended import get_jwt_identity,verify_jwt_in_request
```

Ruff (0.8.2)

`flask_jwt_extended.verify_jwt_in_request` imported but unused

Remove unused import: `flask_jwt_extended.verify_jwt_in_request`

(F401)